CI.hs

```diff
@@ -51,7 +48,15 @@ data StackOptions = StackOptions
     , ghcOptions :: Maybe String -- If 'Just _', pass '--ghc-options="xxx"' to 'stack build' (for ghc verbose, try 'v3').
     } deriving (Show)
 
-data GhcFlavor = Ghc8101 | Ghc881 | Ghc882 | Ghc883 | Da | GhcMaster String
+data GhcFlavor = Ghc8101
+               | Ghc881
+               | Ghc882
+               | Ghc883
+               | Da { mergeBaseSha :: String
```

Comment:
Shouldn't comment on style matters but I think this will fit on a single line eh?

Reply:
I'm fine with whatever style you prefer on this project. I've updated this one (though you need longer-than-80-chars lines for it to fit ;-) ); don't hesitate to point out others. In particular, I have no idea how the `parseOptions` function "should" be indented.

Reply:
"Consistency is the hobgoblin of little minds" 🤣